BiometricLoginKotlin/app/src/main/java/com/example/biometricloginsample/EnableBiometricLoginActivity.kt

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (C) 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License
+ */
+package com.example.biometricloginsample
+
+import android.content.Context
+import android.os.Bundle
+import android.util.Log
+import android.view.inputmethod.EditorInfo
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.AppCompatButton
+import androidx.appcompat.widget.AppCompatEditText
+import androidx.biometric.BiometricManager
+import androidx.biometric.BiometricPrompt
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+
+class EnableBiometricLoginActivity : AppCompatActivity() {
+
+    private val TAG = "EnableBiometricLogin"
+    private lateinit var loginViewModel: LoginWithPasswordViewModel
+    private lateinit var cryptographyManager: CryptographyManager
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_enable_biometric_login)
```

Comment:
Since Android Studio 3.6 is stable, and this is a new sample, I think we should use [view binding](https://developer.android.com/topic/libraries/view-binding) here, rather than `findViewById`.

Reply:
View binding is one more thing for devs to learn. Maybe in the future when adoption of the Biometric library is much higher we can do that. For now everyone knows how to findViewById -- this way they can focus on the biometrics and cryptographic stuff. But definitely as a future feature

Reply:
I can appreciate that you want to keep the sample as simple for developers as possible, but `findViewById` is outdated and not typesafe, whereas view binding is:

- fast
- nullsafe
- typesafe

It is also no more (and often _less_) code than using `findViewById`.

So yes, please switch to using view binding.

Reply:
There are also one more bits about View binding that I meant to add but had to run last night: 

1. It's not used by 1P apps
2. It has some known performance issues. 
3. There is some parallel work to verify that large apps find it competitive and based on the results a decision will be made by IO

And since this sample is also targeting government/financial/healthcare/enterprise apps, that's why I made the decision to forego view binding until the result of that parallel work comes out.

Reply:
(I am not attached one way or the other. I'm just sharing the thought process that lead to the decision. So if you or someone else feels strongly about view binding, I'll switch to it)